spec/services/slack/messengers/potential_spammer_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe Slack::Messengers::PotentialSpammer, type: :service do
+  let_it_be_readonly(:user) { create(:user) }
+
+  let(:default_params) do
+    {
+      user: user
+    }
+  end
```

Comment:
block syntax feels a bit heavyweight here

Reply:
haha i actually thought about it and forgot to change it, thanks!